tldw_Server_API/app/core/Workflows/adapters/media/ingest.py

```
@@ -190,27 +194,30 @@ async def run_media_ingest_adapter(config: dict[str, Any], context: dict[str, An
-                        content=extracted_text,
-                        keywords=keywords,
-                        overwrite=False,
-                        chunk_options=None,
-                        chunks=None,
-                    )
+                    with managed_media_database(
+                        "workflow_engine",
+                        db_path=_mdb_path,
+                        initialize=False,
+                    ) as mdb:
+                        media_id, media_uuid, msg = get_media_repository(mdb).add_media_with_keywords(
```

Ruff (0.15.6)

[warning] Unpacked variable `media_uuid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)